src/Transports/MassTransit.SqlTransport.PostgreSql/SqlTransport/PostgreSql/PostgresDatabaseMigrator.cs

```diff
@@ -1237,6 +1237,11 @@ async Task GrantAccess(IPostgresSqlTransportConnection connection, SqlTransportO
             }
 
             var principal = PostgresSqlTransportConnection.GetAdminMigrationPrincipal(options);
+            if (options.Role != principal)
```

Comment:
Is this case sensitive? If not, shouldn't this be an insensitive comparison?

Reply:
`options.Role` and `principal` are used as a quoted parameters, so they are case sensitive. Here's the [reference](https://www.postgresql.org/docs/9.4/sql-syntax-lexical.html#SQL-SYNTAX-IDENTIFIERS:~:text=Quoting%20an%20identifier%20also%20makes%20it%20case%2Dsensitive%2C%20whereas%20unquoted%20names%20are%20always%20folded%20to%20lower%20case.)
Accordint to that the comparison must be case sensitive as well: `!string.Equals(options.Role, principal, StringComparison.Ordinal)`. Culture-agnostic and non-linguistic `StringComparison.Ordinal` will fit perfectly in that case.